core/src/jsMain/kotlin/Central.kt

```diff
@@ -16,7 +16,8 @@ public class JsCentral internal constructor(
     private val bluetooth: Bluetooth? = js("window.navigator.bluetooth")
 
     public override fun scanner(): Scanner {
-        TODO("Not yet implemented")
+        bluetooth ?: error("Bluetooth unavailable")
```

Comment:
Nitpick: I think this would be slightly more idiomatic here.

```suggestion
        checkNotNull(bluetooth) { "Bluetooth unavailable" }
```

Reply:
Skipping as this file is under heavy refactor in #24

Reply:
Applied change in #24 under commit 8b27943ede4322db2ba71a5d792517a30c782788.